src/main/java/net/sf/rails/common/notify/Discord.java

```diff
@@ -84,18 +90,21 @@ public void update(String text) {
         public Observable getObservable() {
             return pm.getCurrentPlayerModel();
         }
-    }
 
-    public static void notifyOfGameInit(final RailsRoot root) {
-        instance.init(root);
+        public Player getFormerPlayer() {
+            return formerCurrentPlayer;
```

Comment:
hm formerCurrentPlayer (sounds logically but on the other side horrific :))

Reply:
Haha, yes, yes it is quite ugly. And to be sure, it is not my invention, see SoundEventInterpreter :)